index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <style>
+      #navbar{
+        position: fixed
+        
+      }
+
+  @media (max-width: 600px) {
+  .facet_sidebar {
+    display: none;
+  }
+}
+    </style>
+    <meta charset="UTF-8">
```

Comment:
Please add meta view-port element.

Reply:
- [Optional] It would be nice to explain the importance of having a meta view-port element and it's always a good idea to keep your tone suggestive rather than instructive or authoritative Cheers 👍